### ISSUE TITLE
The block my_external_backup_courses does not define the standard capability block/my_external_backup_courses:myaddinstance

### ISSUE THREAD

**Issue by mudrd8mz** [cperves/moodle-block-my_external_privatefiles#5]
When testing in Moodle 2.6 (which is the lowest supported Moodle version now we accept new plugins for), I am getting

```
The block my_external_backup_courses does not define the standard capability
block/my_external_backup_courses:myaddinstance
```

This capability has been expected since 2.4 though. See blocks/upgrade.txt file

**Comment by cperves** [comment]
Working on 2.7 migration, so I think that I'll deliver the next corrected version in  2.7

**Comment by cperves** [comment]
made in the latest version tested on 2.9.2 moodle version
